config/routes.rb

```diff
@@ -0,0 +1,18 @@
+Rails.application.routes.draw do
+  root to: "homepages#index"
+  get "homepages/index"
+  resources :works
```

Comment:
Actually, since you do nested routes with `works` below, you can remove this line and your tests still pass!